src/components/Calculator.js

```diff
@@ -1,47 +1,59 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: null,
+      next: null,
+    };
+    this.handler = this.handler.bind(this);
+  }
+
+  handler(e) {
+    this.setState((state) => calculate(state, e.target.innerText));
```

Comment:
- [ ] The handler is failing while doing an operation on the `divide`.

![Capture web_19-10-2021_151428_localhost](https://user-images.githubusercontent.com/61562626/137928754-015f537c-a046-4b48-94ca-72829d7fdafb.jpeg)